sdk/keyvault/azkeys/client.go

```diff
@@ -284,15 +287,36 @@ type CreateRSAKeyOptions struct {
 
 	// Application specific metadata in the form of key-value pairs.
 	Tags map[string]string `json:"tags,omitempty"`
+
+	// Elliptic curve name. For valid values, see JsonWebKeyCurveName.
+	Curve *JSONWebKeyCurveName `json:"crv,omitempty"`
```

Comment:
Just `KeyCurveName` is what the other SDKs use.

Reply:
Updated